addons/arrays/fnc_sortNestedArray.sqf

```diff
@@ -22,20 +22,23 @@ Returns:
     Sorted array <ARRAY>
 
 Author:
-    commy2
+    commy2, mharis001
 ---------------------------------------------------------------------------- */
 SCRIPT(sortNestedArray);
 
 params [["_array", [], [[]]], ["_index", 0, [0]], ["_order", true, [false]]];
 
-private _helperArray = _array apply {
-    [_x select _index, _x]
-};
+private _helperArray = [];
+private _coefficient = [-1, 1] select _order;
+
+{
+    _helperArray pushBack [_x select _index, _coefficient * _forEachIndex, _x];
+} forEach _array;
```

Comment:
Please keep the `apply`  to save N `pushBack` commands.
```sqf
private _helperArray = _array apply {
    [_x select _index, _coefficient * _forEachIndex, _x]
};
```

Reply:
no `_forEachIndex` in apply

Reply:
Dang.